package.json

```diff
@@ -55,6 +55,9 @@
     "karma-sourcemap-loader": "0.3.7",
     "karma-spec-reporter": "0.0.31",
     "karma-webpack": "3.0.0",
+    "lodash": "^4.17.11",
+    "mime": "^2.4.0",
+    "parsejson": "0.0.3",
```

Comment:
We don't use them directly, but I am trying to fix the github security warning alerts. In reality, the real fix is to upgrade the packages we do use, but that requires more time I can put on it right now:

https://github.com/iotile/ng-iotile-cloud/network/alerts

Reply:
oh ok! PR looks good then!

Reply:
could you remove these 2 lines of code? they use the old version of observable and break the new library
https://github.com/iotile/ng-iotile-cloud/blob/master/src/cloud.service.ts#L6
https://github.com/iotile/ng-iotile-cloud/blob/master/src/cloud.service.ts#L7

Reply:
What do you mean be remove? They are used. How do I remove them? Can I just upgrade the rs library?

Reply:
If you just delete those two lines of code, the library has already been updated and `ForkJoin` has already been imported on line 5 and `of` is not being used.
These two lines don't break the webapp because it includes the package `rxjs-compat` for backward compatibility, but that package is not included in the new library